{
    "sorter_name": "herdingspikes",
    "sorter_version": "0.3.99",
    "datetime": "2022-03-19T21:56:25.510532",
    "runtime_trace": [],
    "error": true,
    "error_trace": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.9/site-packages/spikeinterface/sorters/basesorter.py\", line 200, in run_from_folder\n    SorterClass._run_from_folder(output_folder, sorter_params, verbose)\n  File \"/usr/local/lib/python3.9/site-packages/spikeinterface/sorters/herdingspikes/herdingspikes.py\", line 170, in _run_from_folder\n    recording = st.normalize_by_quantile(\n  File \"/usr/local/lib/python3.9/site-packages/spikeinterface/toolkit/preprocessing/normalize_scale.py\", line 175, in normalize_by_quantile\n    return NormalizeByQuantileRecording(*args, **kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/spikeinterface/toolkit/preprocessing/normalize_scale.py\", line 55, in __init__\n    random_data = get_random_data_chunks(recording, **random_chunk_kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/spikeinterface/toolkit/utils.py\", line 39, in get_random_data_chunks\n    chunk = recording.get_traces(start_frame=start_frame,\n  File \"/usr/local/lib/python3.9/site-packages/spikeinterface/core/baserecording.py\", line 101, in get_traces\n    traces = rs.get_traces(start_frame=start_frame, end_frame=end_frame, channel_indices=channel_indices)\n  File \"/usr/local/lib/python3.9/site-packages/spikeinterface/toolkit/preprocessing/filter.py\", line 107, in get_traces\n    traces_chunk, left_margin, right_margin = get_chunk_with_margin(self.parent_recording_segment,\n  File \"/usr/local/lib/python3.9/site-packages/spikeinterface/toolkit/preprocessing/tools.py\", line 31, in get_chunk_with_margin\n    traces_chunk = rec_segment.get_traces(start_frame - left_margin, end_frame + right_margin, channel_indices)\n  File \"/usr/local/lib/python3.9/site-packages/spikeinterface/core/frameslicerecording.py\", line 74, in get_traces\n    traces = self._parent_recording_segment.get_traces(start_frame=parent_start,\n  File \"/usr/local/lib/python3.9/site-packages/spikeinterface/extractors/nwbextractors.py\", line 223, in get_traces\n    traces = es.data[start_frame:end_frame, channel_indices]\n  File \"h5py/_objects.pyx\", line 54, in h5py._objects.with_phil.wrapper\n  File \"h5py/_objects.pyx\", line 55, in h5py._objects.with_phil.wrapper\n  File \"/usr/local/lib/python3.9/site-packages/h5py/_hl/dataset.py\", line 708, in __getitem__\n    if self._fast_read_ok and (new_dtype is None):\n  File \"/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/functools.py\", line 969, in __get__\n    val = self.func(instance)\n  File \"/usr/local/lib/python3.9/site-packages/h5py/_hl/dataset.py\", line 687, in _fast_read_ok\n    self._extent_type == h5s.SIMPLE\n  File \"/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/functools.py\", line 969, in __get__\n    val = self.func(instance)\n  File \"h5py/_objects.pyx\", line 54, in h5py._objects.with_phil.wrapper\n  File \"h5py/_objects.pyx\", line 55, in h5py._objects.with_phil.wrapper\n  File \"/usr/local/lib/python3.9/site-packages/h5py/_hl/dataset.py\", line 576, in _extent_type\n    return self.id.get_space().get_simple_extent_type()\n  File \"h5py/_objects.pyx\", line 54, in h5py._objects.with_phil.wrapper\n  File \"h5py/_objects.pyx\", line 55, in h5py._objects.with_phil.wrapper\n  File \"h5py/h5d.pyx\", line 299, in h5py.h5d.DatasetID.get_space\nValueError: Invalid dataset identifier (invalid dataset identifier)\n",
    "run_time": null
}